{"name": "test_incorrect[incorrect_data1]", "status": "passed", "attachments": [{"name": "log", "source": "eb25e988-bf81-4683-88d2-3f9047fa18fa-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "incorrect_data", "value": "{'username': '', 'password': 'nmb_python', 'check': '账号不能为空'}"}], "start": 1594727188248, "stop": 1594727240434, "uuid": "4965e679-f974-4615-a77d-455cbca57837", "historyId": "0afd9caeb763ed1e2e2a9e58757ef7f4", "testCaseId": "486418aa0944ed0ecb75b40c2b58e369", "fullName": "TestCases.test_login.TestLogin#test_incorrect", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KFH4O7N"}, {"name": "thread", "value": "10824-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}